demos/requirements.txt

```diff
@@ -8,3 +8,6 @@ tokenizers
 tensorboard
 tensorboardX
 tqdm
+jupyterlab
+ipywidgets
+opencv-python-headless
```

Comment:
Please remove OpenCV from here, as well. Also, keep the list in alphabetic order.

Reply:
You also need to regenerate `ci/requirements-demos.txt` from the new version of this file.